### PR TITLE
Add a summary of the project in the doc

### PR DESCRIPTION
**Pull request by pierrepo**
- [x] Add / update the documentation

Solve #151

### CODE REVIEW

**Comment by pierrepo** [comment]
@jtpio I wrote a little summary text of the project.
Tell me if it's clear enough.

File: docs/index.rst [hunk 1]
```diff
@@ -3,6 +3,23 @@
    :width: 100%
    :align: center
 
+
+Plasma, aka in French “Plateforme d’eLearning pour l’Analyse de données Scientifiques MAssives”, aims at creating
```
Comment (jtpio) [inline docs/index.rst:7]: Is it "eLearning" or "e-Learning"?

```suggestion
Plasma, aka in French “Plateforme d’e-Learning pour l’Analyse de données Scientifiques MAssives”, aims at creating 
```

File: docs/index.rst [hunk 2]
```diff
@@ -3,6 +3,23 @@
    :width: 100%
    :align: center
 
+
+Plasma, aka in French “Plateforme d’eLearning pour l’Analyse de données Scientifiques MAssives”, aims at creating 
+an interactive tool to teach computational analysis of massive scientific data. 
+Plasma was born out of the need to offer a reproducible and high-performance analysis environment to our students.
+
+Based on the Jupyter ecosystem, Plasma allows the creation and the management of isolated and highly customisable environments,
```
Comment (jtpio) [inline docs/index.rst:11]: We usually keep the US spelling in the docs and repo:

```suggestion
Based on the Jupyter ecosystem, Plasma allows the creation and the management of isolated and highly customizable environments, 
```

File: docs/index.rst [hunk 3]
```diff
@@ -3,6 +3,23 @@
...
+an interactive tool to teach computational analysis of massive scientific data. 
+Plasma was born out of the need to offer a reproducible and high-performance analysis environment to our students.
+
+Based on the Jupyter ecosystem, Plasma allows the creation and the management of isolated and highly customisable environments, 
+with an easy deployement on bare-metal servers or virtual machines.
+
+Plasma utilizes `tljh-repo2docker <https://github.com/plasmabio/tljh-repo2docker>`_, 
+a `repo2docker <https://github.com/jupyter/repo2docker>`_ plugin for `The Littlest JupyterHub <https://tljh.jupyter.org/en/latest/>`_.
+
+For more details, have a look to:
```
Comment (jtpio) [inline docs/index.rst:17]:
```suggestion
For more details, have a look at:
```

**Comment by jtpio** [comment]
Looks good, thanks @pierrepo!

Left a couple of minor inline comments.

File: docs/index.rst (hunk 1 above)
Comment (jtpio) [reply docs/index.rst:7]: It's also e-Learning in the logo.

**Comment by pierrepo** [comment]
Thanks for the edits!
Sounds very good to me @jtpio

**Pull request merged**